storage/system-test/acl.test.js

```diff
@@ -16,51 +16,43 @@
 'use strict';
 
 const storage = require(`@google-cloud/storage`)();
-const uuid = require(`node-uuid`);
+const uuid = require(`uuid`);
 const path = require(`path`);
 const run = require(`../../utils`).run;
 
 const cwd = path.join(__dirname, `..`);
 const bucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
+const bucket = storage.bucket(bucketName);
 const userEmail = `00b4903a973860a50828a620e09dde96aae6122ca4f0835bd469384659f1a5b8`;
 const fileName = `test.txt`;
 const filePath = path.join(__dirname, `../resources`, fileName);
 const cmd = `node acl.js`;
 
 describe('storage:acl', () => {
-  before((done) => {
-    storage.createBucket(bucketName, (err, bucket) => {
-      assert.ifError(err);
-      bucket.upload(filePath, done);
-    });
-  });
+  before(() => bucket.create().then((results) => results[0].upload(filePath)));
 
-  after((done) => {
-    storage.bucket(bucketName).file(fileName).delete(() => {
-      // Ignore error
-      setTimeout(() => {
-        storage.bucket(bucketName).delete(() => {
-          // Ignore error
-          done();
-        });
-      }, 2000);
-    });
+  after(() => {
+    // Try deleting all files twice, just to make sure. Ignore any errors.
```

Comment:
The implementation of `Bucket#deleteFiles` itself is sketchy, and it doesn't always work, causing our tests to randomly fail. Just fighting sketch with sketch.